exec/cnex/lib/sqlite.c

```diff
@@ -0,0 +1,404 @@
+#ifdef _WIN32
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
I'm curious about the placement of this `#define`. Is this something that needs to be defined before including `sqlite.h`? Or is it something that just relates to the code within this .cpp file? I suspect that it is the latter, in which case it should be moved *below* all the includes. That makes sure that the define only affects *this* source file, and not the included headers.

Reply:
Yea, this is really only for the object::toString() function.  So, I moved it to after all of the includes.